docs/UserGuide.md

```diff
@@ -251,6 +251,122 @@ Examples:
 * `deletetask 2` deletes the task with index 2 from the task list.
 
 
+### Grading an assignment: `grade`
+Grades a student’s assignment.
+
+Format: `grade n/STUDENT_NAME a/ASSIGNMENT g/GRADE`
+
+Parameters:
+* STUDENT_NAME The name of the student.
+* ASSIGNMENT The name of the assignment.
+* GRADE The grade of the student.
```

Comment:
Hmm, this one we might want to discuss as a team for the wording.
In CS1101S context, its XP, not grade.
Might want to update this to tailor to our target audience